e2e-tests/development-runtime/cypress/integration/eslint-rules/limited-exports-page-templates.js

```diff
@@ -1,17 +1,14 @@
 describe(`limited-exports-page-templates`, () => {
-  // Skipped because HMR not show warnings because of https://github.com/webpack-contrib/webpack-hot-middleware/pull/397
-  it.skip(`should log warning to console for invalid export`, () => {
-    cy.visit(`/eslint-rules/limited-exports-page-templates`, {
-      onBeforeLoad(win) {
-        cy.stub(win.console, "log").as(`consoleLog`)
-      },
-    }).waitForRouteChange()
+  it(`should log warning to console for invalid export`, () => {
+    cy.visit(
+      `/eslint-rules/limited-exports-page-templates`
+    ).waitForRouteChange()
```

Comment:
We were getting `Attempted to wrap log which is already wrapped` error when trying to stub or spy on console.log here because all `visit` calls already spy/wrap it ( https://github.com/gatsbyjs/gatsby/blob/b2ae4da089d2783dc83738224f362e6f8fb44d0c/e2e-tests/development-runtime/cypress/support/commands.js#L88-L102 ).